suites/reef/nvmeof/tier-1_nvmeof_functional_Regression.yaml

```diff
@@ -327,6 +327,23 @@ tests:
       name: Reboot client node and check NVMe namespaces
       polarion-id: CEPH-83576087
 
+  - test:
+      abort-on-fail: false
+      config:
+        gw_node: node6
+        rbd_pool: rbd
+        do_not_create_image: true
+        rep-pool-only: true
+        rep_pool_config:
```

Comment:
Will it make sense if we specify the original size  and expand, shrink size given from this suite file so we know what is expectation vs what was it before the test and compare with logs instead of going to test module?